config-default.yml

```diff
@@ -175,6 +175,7 @@ guild:
         esoteric:                           470884583684964352
         verification:                       352442727016693763
         voice_gate:                         764802555427029012
+        how-to-get-help:                    704250143020417084
```

Comment:
* This isn't being used anywhere.
* Any added constants should have an analogue in `constants.py`
* This doesn't follow the naming convention of using underscores.